Mazes_from_video/playvideo_and_getcoordinate.py

```diff
@@ -0,0 +1,252 @@
+            else:
+                image = img
+                #image = cv2.resize(img,(width,hight))
+                #image = cv2.resize(img,(width,hight)) # set stand size is 640 * 480.
+            # To improve performance, optionally mark the image as not writeable to
+            # pass by reference.
+            
+            
+            image.flags.writeable = True
+            image = cv2.cvtColor(image, cv2.COLOR_BGR2RGB)
+            results = pose.process(image)
+            size = image.shape   # 取得攝影機影像尺寸
+            w = size[1]        # 取得畫面寬度
+            h = size[0]        # 取得畫面高度
+            #print("w:"+str(w)+" h:"+str(h))
+            if results.pose_landmarks:
+                #right hand
+                rx = zero_to_one(results.pose_landmarks.landmark[20].x)
+                ry = zero_to_one(results.pose_landmarks.landmark[20].y)
+                right_hand.append([int(rx *w),int(ry *h)])
+                ##left hand
+                #left_hand.append([int(results.pose_landmarks.landmark[19].x *w),int(results.pose_landmarks.landmark[19].y *h)])
+                ##right foot
+                #right_foot.append([int(results.pose_landmarks.landmark[32].x *w),int(results.pose_landmarks.landmark[32].y *h)])
+                ##left foot
+                #left_foot.append([int(results.pose_landmarks.landmark[31].x *w),int(results.pose_landmarks.landmark[31].y *h)])
+
+            # Draw the pose annotation on the image.
+            image.flags.writeable = True
+            image = cv2.cvtColor(image, cv2.COLOR_RGB2BGR)
+            mp_drawing.draw_landmarks(
+                image,
+                results.pose_landmarks,
+                mp_pose.POSE_CONNECTIONS,
+                landmark_drawing_spec=mp_drawing_styles.get_default_pose_landmarks_style()) 
+            image = draw_circle(img,right_hand,-1)
+            image = draw_dot(image,right_hand)
+            # Flip the image horizontally for a selfie-view display.
+            cv2.namedWindow('MediaPipe Pose', 0)
+            cv2.resizeWindow('MediaPipe Pose', width,hight)
+            cv2.imshow('MediaPipe Pose', image)
+            if cv2.waitKey(5) & 0xFF == 27:
+                break
+    
+#right_hand,binarization_arr=get_track("test1.mp4")
+#print("successful end.")
+#print("right_hand")
+#print(right_hand)
+#print("binarization_arr")
+#for i in range(hight):
+#    for j in range(width):
+#        if (binarization_arr[1][j][i] == 1):
+#            print("x:" + str(j) + " y:" + str(i))
+#print("game over.")
+#print(len(binarization_arr))
+#print(len(right_hand))
+#print(len(binarization_arr[0][1]))
+
+#path = 'output.txt'
+#f = open(path, 'w')
+#for h in range (hight+5):
+#    for w in range (width+5):
+#        for i in range(len(binarization_arr)):
+#            k = 0
+#            if binarization_arr[i][w][h] == 1:
+#                k = 1
+#                break
+#        print(str(k),end='', file=f)
+#    print('', file=f)
+#f.close()
```

Comment:
Remove this block if it'is not needed, otherwise add some descriptions about using it.